sdk/devcenter/azure-developer-devcenter/swagger/README.md

```diff
@@ -1,7 +1,7 @@
 ## Generate autorest code
 
 ```yaml
-use: '@autorest/java@4.1.5'
+use: '@autorest/java@4.1.7'
 input-file:
   - https://github.com/Azure/azure-rest-api-specs/blob/75a8d8dcc9f6d0ec626bdeb32f5154f20c8c61cd/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-03-01-preview/devcenter.json
   - https://github.com/Azure/azure-rest-api-specs/blob/75a8d8dcc9f6d0ec626bdeb32f5154f20c8c61cd/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-03-01-preview/devbox.json
```

Comment:
Please update it to commit id on main branch.

Reply:
Good catch. Updated to our most recent checkin on main.